src/extension.ts

```diff
@@ -66,7 +67,7 @@ class CatCodingPanel {
         const panel = vscode.window.createWebviewPanel(
             CatCodingPanel.viewType,
             'Cat Coding',
```

Comment:
At some point we should rename all the :cat2:  :cat2:  :sweat_smile:

Reply:
Yeah, let's rename them before merging to `master`...